tests/CoreTest.php

```diff
@@ -179,11 +179,11 @@ public function testFindRevisionHistory(): void
         $this->assertCount(2, $revisions);
         $this->assertContainsOnly(Revision::class, $revisions);
 
-        $this->assertSame('2', $revisions[0]->getRev());
+        $this->assertSame('2', (string) $revisions[0]->getRev());
```

Comment:
It's not the same type depending on the db ?

Reply:
No. PostgreSQL is returning integers.